panel/tests/pane/test_vega.py

```diff
@@ -230,7 +230,7 @@ def test_altair_pane(document, comm):
     chart.mark = 'point'
     chart.data.values[0]['x'] = 'C'
     pane.object = chart
-    point_example = dict(vega_example, mark='point')
+    point_example = dict(vega_example, data={},  mark='point')
```

Comment:
This change is needed to be able to run this test in parallel, as it seems to be depends on the previous tests in this file have been run before it. I don't know why this is the case. 

E.g. this fails `pytest panel/tests -k "altair"` but this does not `pytest panel/tests -k "altair or vega"` before this change.

``` python-traceback
============================================== FAILURES =====================================================
_________________________________________________ test_altair_pane _________________________________________________

document = <bokeh.document.document.Document object at 0x7f1318570b80>
comm = Comm(id='bb34c97071754b02b7db8bc3f4a895d3', name='Comm01263')

    @altair_available
    def test_altair_pane(document, comm):
        pane = pn.panel(altair_example())
    
        # Create pane
        model = pane.get_root(document, comm=comm)
        assert isinstance(model, VegaPlot)
    
        expected = dict(vega_example, data={})
        if altair_version >= Version('4.0.0'):
            expected['config'] = vega4_config
        assert dict(model.data, **blank_schema) == dict(expected, **blank_schema)
    
        cds_data = model.data_sources['data'].data
        assert np.array_equal(cds_data['x'], np.array(['A', 'B', 'C', 'D', 'E']))
        assert np.array_equal(cds_data['y'], np.array([5, 3, 6, 7, 2]))
    
        chart = altair_example()
        chart.mark = 'point'
        chart.data.values[0]['x'] = 'C'
        pane.object = chart
        point_example = dict(vega_example, mark='point')
        if altair_version >= Version('4.0.0'):
            point_example['config'] = vega4_config
>       assert dict(model.data, **blank_schema) == dict(point_example, **blank_schema)
E       AssertionError: assert {'$schema': '...ative'}}, ...} == {'$schema': '...ative'}}, ...}
E         Omitting 4 identical items, use -vv to show
E         Differing items:
E         {'data': {}} != {'data': {'values': [{'x': 'A', 'y': 5}, {'x': 'B', 'y': 3}, {'x': 'C', 'y': 6}, {'x': 'D', 'y': 7}, {'x': 'E', 'y': 2}]}}
E         Full diff:
E           {
E            '$schema': '',
E            'config': {'view': {'continuousHeight': 300,...
E         
E         ...Full output truncated (19 lines hidden), use '-vv' to show

panel/tests/pane/test_vega.py:236: AssertionError
============================================= short test summary info ==============================================
FAILED panel/tests/pane/test_vega.py::test_altair_pane - AssertionError: assert {'$schema': '...ative'}}, ...} == {'$schema': '...ative'}}, ...}

Reply:
Hmm, is the Vega pane modifying the data inplace somehow. Based on a quick scan I couldn't find anything that could cause this.

Reply:
I think the `data` from `point_example` should be empty. It does not make any sense that it has `data.values[0]['x'] = 'A'` when we a couple of lines above changed it to be `C`.  

The problem is because of a `pop` in `test_vega_pane` will push a fix.